Core/src/Testing/FileListFilterIterator.php

```diff
@@ -17,6 +17,8 @@
 
 namespace Google\Cloud\Core\Testing;
 
+use Google\Cloud\Dev\DocGenerator\RegexFileFilter;
```

Comment:
Is this used?

Reply:
yes, on line 79. It wasn't imported before as it should have been.

Reply:
fixed.